kola/tests/misc/state.go

```diff
@@ -0,0 +1,62 @@
+// Copyright The Mantle Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package misc
+
+import (
+	"fmt"
+
+	"github.com/coreos/go-semver/semver"
+
+	"github.com/flatcar/mantle/kola/cluster"
+	"github.com/flatcar/mantle/kola/register"
+)
+
+func init() {
+	register.Register(&register.Test{
+		Run:         OverlayCleanup,
+		ClusterSize: 1,
+		Name:        "cl.overlay.cleanup",
+		Distros:     []string{"cl"},
+		MinVersion:  semver.Version{Major: 3511},
```

Comment:
Should the minversion be higher?

```suggestion
		MinVersion:  semver.Version{Major: 3530},
```